Pierre Flammer: Added VolumeGroup functionality

Pierre-PvF opened the pull request:
# Description

Please describe your pull request in detail.

## Related Issue

This project only accepts pull requests related to open issues.

* If suggesting a new feature or change, please discuss it in an issue first.
* If fixing a bug, there should be an issue describing it with steps to reproduce

Fixes #228 

## Motivation and Context

 VolumeGroup functionality was missing in our powershell toolkit

## How Has This Been Tested?

Did lots of test in my homelab. 

## Screenshots (if appropriate):

jaapbrasser commented:
Hey @Pierre-PvF, I noticed you didn't update the Rubrik.psd1, so the new functions are not loaded. Could you update line 105 in Rubrik.psd to this:
               'Sync-RubrikTag', 'Get-RubrikVolumeGroup', 'Get-RubrikVolumeGroupMount',
               'New-RubrikVolumeGroupMount', 'Remove-RubrikVolumeGroupMount')

Then we should be good to go, I tried out the functions and they seemed to be working fine after doing that!

Pierre-PvF commented:
Ah shit... Yeah forgot that file... Will do that asap.

Thanks.

On Thu, Jan 10, 2019, 12:36 Jaap Brasser <notifications@github.com wrote:

> Hey @Pierre-PvF <https://github.com/Pierre-PvF>, I noticed you didn't
> update the Rubrik.psd1, so the new functions are not loaded. Could you
> update line 105 in Rubrik.psd to this:
> 'Sync-RubrikTag', 'Get-RubrikVolumeGroup', 'Get-RubrikVolumeGroupMount',
> 'New-RubrikVolumeGroupMount', 'Remove-RubrikVolumeGroupMount')
>
> Then we should be good to go, I tried out the functions and they seemed to
> be working fine after doing that!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/rubrikinc/rubrik-sdk-for-powershell/pull/227#issuecomment-453065613>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ArkpWZ-viMyvFmRAcV09Gxh0pSzjs4Pyks5vByW2gaJpZM4Z2_er>
> .
>

Pierre-PvF commented:
Hi Jaap,

I added the missing lines to Rubrik.psd. Also removed the Get-RubrikAPIData.ps1 in the root of the module.
Feel free to change the version number to an appropriate one.

Pierre
